### DISCUSSION TITLE
Emoji tags (i.e. :wrench:) don't render their image on GH app.

### DISCUSSION THREAD

**Discussion by kjkent** [community/community#31355] [Mobile]
Behaviour observed on Pixel 4XL on Android 13.

**Comment by bewuethr** [comment]
I can see this as well, except only for non-Unicode emoji:

![Screenshot_20220919-214554.png](https://user-images.githubusercontent.com/8521043/191149757-33c98851-1ae4-4489-8c35-58bf13b6bf54.png)

**Comment by VDuchauffour** [comment]
Same here, emoji are not displayed on Android app like has below. I use a OnePlus Nord 2 CE.
![Screenshot_2022-12-12-14-48-33-30_320a9a695de7cdce83ed5281148d6f19.jpg](https://user-images.githubusercontent.com/40724343/207062414-e06a812e-3ab9-47ba-a853-9a8cafbc01d0.jpg)

**Reply by Igorko777** [reply]
Android 14
